src/mappings/util/metadata.ts

```diff
@@ -18,7 +19,7 @@ export async function fetchMetadata(url: string) {
         },
         withCredentials: false,
         timeout: 15000,
-        maxRedirects: 1,
+        maxRedirects: 3,
```

Comment:
Have you checked with @Bradez about this? He stated that we should allow at most 1 redirect.
Can you confirm this change @Bradez ?

Reply:
@leonardocustodio @Bradez , there are migrated assets with URLs starting with `http://` that redirect to `https://` with the same uri (another redirect). Therefore, at least 2 redirects are needed for those. I increased it to 3 to be on the safe side

Reply:
I don't want to add unnecessary logic, but could we do 2 redirects if the URI starts with `https://platform.production.enjinusercontent.com/`, otherwise 1 redirect? I want it to conform to the Metadata Standard, but respect that our "hack" somewhat contravenes this. Yet, it was only temporary, and the intention was for people to migrate away. Won't be permanent.